autoload/battery/backend/termux.vim

```diff
@@ -19,7 +19,7 @@ endfunction
 
 function! s:on_exit(backend, buffer, exitval) abort
   let content = join(a:buffer, '')
-  let a:backend.is_charging = json_decode(content).status !=# 'NOT_CHARGING'
+  let a:backend.is_charging = json_decode(content).status !=# 'DISCHARGING'
```

Comment:
Why did you change the term?

Reply:
Because mine said its DISCHARGING instead of NOT_CHARGING![Screenshot_20210404-204238.png](https://user-images.githubusercontent.com/47383486/113510718-4fafa700-9586-11eb-818d-0b3d358bf7fe.png)